gh-docs/message-processor/Admin_Services.md

```diff
@@ -0,0 +1,58 @@
+# How to create admin-services for the Message Processor
```

Comment:
Shall we make the title more generic? Maybe something like "How to create admin-services for EI"

Reply:
Keeping instructions specific to message processor is OK IMO.

Reply:
We could keep it generic, but I might be missing out on some small details that may not apply in all cases.